sandbox/README.md

```diff
@@ -7,3 +7,8 @@ machine.
 
 To learn how to run the sandbox environment, see
 https://www.opencue.io/docs/quick-starts/.
+
+## Loki logging
+
+Too use loki to store logs requires installing the docker drivers. see:
+https://grafana.com/docs/loki/latest/clients/docker-driver/
```

Comment:
I do wonder if this note is a bit hidden away on it's own. I think the instruction to install the loki docker plugin should be added to the quick start [docs](https://github.com/AcademySoftwareFoundation/opencue.io/blob/master/content/docs/Quick%20starts/quick-start-linux.md#deploying-the-opencue-sandbox-environment)  alongside this PR so the guide still works without users needing to look in the repo

Reply:
Instead of making these services part of the sandbox, I'm opting to move them to their own docker-compose file so it is an addition, this will make it less likely for a new user to get tripped up by it.